.github/workflows/ci.yml

```diff
@@ -68,6 +68,12 @@ jobs:
           config: { name: Static DRM, flags: -GNinja -DSFML_USE_DRM=TRUE -DSFML_RUN_DISPLAY_TESTS=FALSE }
         - platform: { name: Linux GCC, os: ubuntu-22.04  }
           config: { name: Shared DRM, flags: -GNinja -DBUILD_SHARED_LIBS=TRUE -DSFML_USE_DRM=TRUE -DSFML_RUN_DISPLAY_TESTS=FALSE }
+        - platform: { name: Linux GCC, os: ubuntu-22.04 }
+          config: { name: OpenGL ES, flags: -GNinja -DSFML_OPENGL_ES=ON -DSFML_RUN_DISPLAY_TESTS=ON }
+          type: { name: Release }
+        - platform: { name: Linux GCC, os: ubuntu-22.04 }
+          config: { name: OpenGL ES, flags: -GNinja -DSFML_OPENGL_ES=ON -DSFML_RUN_DISPLAY_TESTS=ON }
+          type: { name: Debug, flags: -DCMAKE_BUILD_TYPE=Debug -DSFML_ENABLE_COVERAGE=TRUE }
```

Comment:
What do you think about simply adding a new line to the main build matrix for this? that's simpler than writing these 6 lines plus it means we get 4 jobs instead of 2 so better coverage of all possible configs. I'd also like to promote the Windows OpenGL ES jobs and the Linux DRM jobs. It would let us deduplicate a lot of code.

Reply:
We can do that all in a dedicated PR.

Reply:
That works for me